### PR TITLE
Acceptance test to check the versions of an uploaded file

### PR DESCRIPTION
**Pull request by jasson99**
## Description
Acceptance test to check the versions of an uploaded file

## Related Issue

- Fixes <issue_link>

## How Has This Been Tested?

- test environment:

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [x] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] ...

### CODE REVIEW

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>Trashbin</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/phoenix/6196/

<p>


















</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>Trashbin</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/phoenix/6221/

<p>























</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>Files</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/phoenix/6221/

<p>


















</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>Files</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/phoenix/6222/

<p>




</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>SharingPublic</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/phoenix/6241/

<p>


























































</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>Files</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/phoenix/6241/

<p>






































</p></details>

**Comment by jasson99** [comment]
@skshetry @HariBhandari07 @dpakach

File: tests/acceptance/features/webUIFiles/versions.feature [hunk 1]
```diff
@@ -11,83 +11,69 @@ Feature: Versions of a file
...
-    And user "user0" has uploaded file with content "new lorem content" to "/lorem-file.txt"
+    Given user "user0" has uploaded file with content "lorem content" to "lorem-file.txt"
+    And user "user0" has uploaded file with content "lorem" to "lorem-file.txt"
+    And user "user0" has uploaded file with content "new lorem content" to "lorem-file.txt"
     And user "user0" has logged in using the webUI
     And the user has browsed to the files page
     And the administrator has cleared the versions for user "user0"
-    When the user browses directly to display the "versions" details of file "lorem-file.txt" in folder "/"
+    When the user browses to display the "versions" details of file "lorem-file.txt"
     And the versions list should contain 0 entries
```
Comment (HariBhandari07) [inline tests/acceptance/features/webUIFiles/versions.feature]: ```Then the versions list should contain 0 entries```

File: tests/acceptance/features/webUIFiles/versions.feature [hunk 2]
```diff
@@ -11,83 +11,69 @@ Feature: Versions of a file
...
     And the administrator has cleared the versions for user "user0"
-    When the user browses directly to display the "versions" details of file "lorem-file.txt" in folder "/"
+    When the user browses to display the "versions" details of file "lorem-file.txt"
     Then the versions list should contain 0 entries
-    When the user logs out of the webUI
-    And user "user1" logs in using the webUI
-    And the user has browsed to the files page
-    And the user browses directly to display the "versions" details of file "lorem-file.txt" in folder "/"
+    And the user re-logs in as "user1" using the webUI
+    And the user browses to display the "versions" details of file "lorem-file.txt"
```
Comment (HariBhandari07) [inline tests/acceptance/features/webUIFiles/versions.feature]: When step missing

**Comment by jasson99** [comment]
@phil-davis

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>Trashbin</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/phoenix/6250/

<p>

































</p></details>

**Comment by jasson99** [comment]
@dpakach @skshetry @HariBhandari07 @phil-davis

File: tests/acceptance/helpers/webdavHelper.js [hunk 3]
```diff
@@ -216,3 +216,17 @@ exports.getSkeletonFile = function (filename) {
       return decodeURIComponent(body.ocs.data[0].contentUrlEncoded)
     })
 }
+
+exports.uploadFileWithcontent = function (user, content, filename) {
```
Comment (skshetry) [inline tests/acceptance/helpers/webdavHelper.js]: camelCasing
```suggestion
exports.uploadFileWithContent = function (user, content, filename) {
```

**Pull request merged**